test/test-article.js

```diff
@@ -202,21 +198,21 @@ describe('Articles', function () {
             res.should.have.status(200);
             res.should.be.json;
             res.should.have.deep.property('body.SUCCESS');
-            res.body.SUCCESS.should.have.property('uri', nURI); // TODO: something about this line errors !!
```

Comment:
I haven't had a problem with this recently - it definitely succeeds now, so I deleted the TODO.